README.md

```diff
@@ -1,13 +1,7 @@
 # Unified Runtime
 
 [![Build and test](https://github.com/oneapi-src/unified-runtime/actions/workflows/cmake.yml/badge.svg)](https://github.com/oneapi-src/unified-runtime/actions/workflows/cmake.yml)
```

Comment:
these will need updating once we've got the CI situation finalized

Reply:
We could maybe point these at [ur-precommit.yml](https://github.com/intel/llvm/actions/workflows/ur-precommit.yml) in intel/llvm but I'm not sure if that's going to exist in the long run.

Reply:
https://github.com/intel/llvm/actions/workflows/ur-precommit.yml/badge.svg

Reply:
I guess deploy docs and the compute benchmarks jobs are staying put. What about nightly?

Reply:
Staying put for now.